nodes/clusters/nodes-cluster-resource-configure.adoc

```diff
@@ -26,6 +26,11 @@ managing application memory by:
 // assemblies.
 
 include::modules/nodes-cluster-resource-configure-about.adoc[leveloffset=+1]
+[discrete]
+[id="additional-resources_understanding-compute-resources"]
+== Additional resources
+* xref:../../nodes/clusters/nodes-cluster-overcommit.html#nodes-cluster-overcommit-reserving-memory_nodes-cluster-overcommit[Understanding compute resources and containers]
```

Comment:
.html -> .adoc

Reply:
Ahh crap. I had .adoc before, but then I re-copied the link because it wasn't working correct. I knew that...